index.html

```diff
@@ -6,6 +6,7 @@
     <title>Document</title>
     <link rel="stylesheet" href="styles/general.css">
     <link rel="stylesheet" href="styles/variables.css">
+    <link rel="stylesheet" href="./styles/index.css">
```

Comment:
instead of doing the css in main copy paste it in general cause we're gonna use it for every page